src/js/components/Articles/ArticleButtons.scss

```diff
@@ -0,0 +1,93 @@
+.reader-buttons {
+  margin: 0 auto;
+  margin-top: 544px;
+  max-width: 400px;
+  border-radius: 50%;
+}
+.videoplayer-buttons {
+  margin: 0 auto;
+  margin-top: 50px;
+  max-width: 400px;
+  border-radius: 50%;
+}
+
+//@import "compass/css3";
```

Comment:
remove ghost code and also post this to everyone

Reply:
removed, can you please approve it